pytential/symbolic/mappers.py

```diff
@@ -187,6 +232,45 @@ class DependencyMapper(DependencyMapperBase, Collector):
     pass
 
 
+class DOFDescriptorCollector(Collector):
```

Comment:
This one is just added for testing. Should it just be in `test_symbolic.py`? Not just if it's more generally useful..

Reply:
I'm not sure I see the general use case, so maybe yeah, my vote would be to move to `testlib` or so.

Reply:
Got it, moved in the latest version!